### PR TITLE
Feature/clang

### PR DESCRIPTION
**Pull request by mmiesch**
This PR is linked to one in [docker](https://github.com/JCSDA/docker/pull/27) - see there for further comments

### CODE REVIEW

**Review by srherbener — APPROVED** [review]
Tried the whole Clang docker_base, docker builds on my iMac, followed by running docker and building ufo-bundle. All the tests pass.

It will be nice to have automatic testing with Clang compilers. Thanks for putting this feature in!

**Comment by mmiesch** [comment]
I'm going to merge this because it has one approval and it's holding up container development

**Pull request merged**
